notes/lecture25.tex

```diff
@@ -0,0 +1,180 @@
+\item Under what conditions that the single-step Newton method works?
+\item How to carefully update $\epsilon$?
+\end{itemize}
+
+\subsubsection{Newton Decrement}
+To apply the Newton's method, first we need to find out the first-order and second-order derivatives of $f_\epsilon$. Note that
+\begin{eqnarray}
+\triangledown f_\epsilon(x)&=&c-\epsilon \sum_{j=1}^m \dfrac{A_j}{A^T_j x-b}\triangleq c-\epsilon A^T S^{-1}\mathbb{1}\label{eq:grad_f_eps}\\
+\triangledown^2f_\epsilon(x)&=&\epsilon A^TS^{-2}A=\epsilon \sum_{j=1}^m\dfrac{A_j A_j^T}{S_j^2}\label{eq:hessian_f_eps}
+\end{eqnarray}
+where $\mathbb{1}=[1,1,\cdots,1]^T\in\mathbb{R}^{m\times1}$, $S=diag\{A^T_1 x-b, A^T_2 x-b, \cdots, A^T_m x-b\}$.
+
+Then the Newton's update can be applied:
+\begin{equation}
+\bar{x}=x-[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)=x-[\epsilon A^TS^{-2}A]^{-1}\left(c-\epsilon A^T S^{-1}\mathbb{1}\right)\label{eq:Newton_update}
+\end{equation}
+
+Recall that the Newton's method finds the solution by making the first-order condition zero. To measure how much the Newton update will decrease the first-order approximation, we introduce the concept of Newton decrement.
+
+Define the Newton decrement as
+\begin{equation}
+q^2(x, \epsilon) = \triangledown f^T_\epsilon(x)[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)\label{eq:newton_decrement}
+\end{equation}
+or $q(x, \epsilon) = [\triangledown^2f_\epsilon(x)]^{-1/2}\triangledown f_\epsilon(x)$. 
+
+Note that the Newton decrement also relates to the difference between $f_\epsilon(x)$ and the minimum of its second-order approximation:
+\begin{eqnarray}
+&&f_\epsilon(x)-\min_{\bar{x}}\left(f_\epsilon(x)+\triangledown f_\epsilon(x)^T(\bar{x}-x)+(\bar{x}-x)^T\triangledown^2 f_\epsilon(x)(\bar{x}-x)\right)\nonumber\\
+&=&f_\epsilon(x)-\left(f_\epsilon(x)-\dfrac{1}{2}\triangledown f^T_\epsilon(x)[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)\right)\nonumber\\
+&=&\dfrac{1}{2}\triangledown f^T_\epsilon(x)[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)\triangleq \dfrac{1}{2}q^2(x, \epsilon).
+\end{eqnarray}
+
+We now will use the Newton decrement to answer the above two questions: conditions for single-step Newton to work and rules for updating $\epsilon$.
+
+\subsubsection{Propositions, Algorithms and Convergence Theorem}
+\begin{proposition}\label{prop1}
+Assume $Ax > b$ and $\|q(x, \epsilon)\|<1$, then we have
+\begin{align}
+c^Tx-c^Tx^{*}\leq 2\epsilon n.
+\end{align}
+\end{proposition}
+In particular, if we maintain that $x_t$ is interior point satisfying $Ax_t>b$,
+and $q(x_t, \epsilon_t) < 1$, then $c^{T}x_t$ converges to $c^Tx^*$ as $\epsilon_t$ goes to $0$, i.e., $x_t$ converges to global optimum. However, the condition $q(x_t,\epsilon_t)<1$ is not trivial.  
+
+\begin{proposition}\label{prop2}
+If $Ax>b$, and $q(x,\epsilon)<1$, then the pure Newton iterate
+step $\bar{x}$ satisfies,
+\begin{align}
+\|q(\bar{x}, \epsilon)\| &\leq \|q(x, \epsilon)\|^2 
```

Comment:
Note that you defined q(x, epsilon) as as scalar function. So you don't need the norms here and elsewhere.